lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -99,6 +99,11 @@ div.course-wrapper {
     padding: ($baseline*2);
     line-height: 1.6;
 
+    .msg-title {
+        font-weight: bold;
+        margin-bottom: 10px;
```

Comment:
NIT: property order

We like to write our styles "out to in". Let's move the `margin-bottom` property above the `font-weight`.